simulator/opendc-experiments/opendc-experiments-allocateam/run-experiments.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
This script is not necessary as there is already a better version (`tools/run.sh`) that first checks whether there is a data folder and aborts the execution if it exists.